components/host-dashboard/CollectiveSettingsModal.js

```diff
@@ -53,6 +53,15 @@ const editAccountSettingsMutation = gqlV2/* GraphQL */ `
         hostFeesStructure
         hostFeePercent
       }
+      childrenAccounts {
```

Comment:
By fetching children accounts for the updated collective, we're making sure that their new fees structure is reflected in the GraphQL cache